Updates to hybrid nodes add-on docs

erchpm opened the pull request:
Add Community add-ons table and metrics-server. Clarify routability requirements for add-ons that run webhooks.

*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

erchpm committed:
Update hybrid-nodes-add-ons.adoc for metrics-server

Add Community add-ons table and metrics-server. Clarify routability requirements for add-ons that run webhooks.

fincd-aws commented on latest/ug/nodes/hybrid-nodes-add-ons.adoc:
```diff
@@ -11,13 +11,13 @@ include::../attributes.txt[]
 [cols="1,1", options="header"]
 |===
-|EKS add-on
+|AWS Add-on
```
Can't have "AWS" for branding reasons.

pgasca commented on latest/ug/nodes/hybrid-nodes-add-ons.adoc:
```diff
@@ -11,13 +11,13 @@ include::../attributes.txt[]
 --
 
-This page describes considerations for running Amazon EKS add-ons from {aws} on Amazon EKS Hybrid Nodes. To learn more about the Amazon EKS add-ons from {aws} and the processes for creating, upgrading, and removing add-ons from your cluster, see <<eks-add-ons>>. The processes for creating, upgrading, and removing Amazon EKS add-ons is the same for Amazon EKS clusters with hybrid nodes as it is fo ...
+This page describes considerations for running {aws} Add-ons and Community add-ons on Amazon EKS Hybrid Nodes. To learn more about Amazon EKS add-ons and the processes for creating, upgrading, and removing add-ons from your cluster, see <<eks-add-ons>>. Unless otherwise noted on this page, the processes for creating, upgrading, and removing Amazon EKS add-ons is the same for Amazon EKS clusters w ...
```
GLOBAL: Don't capitalize "Add-ons" and "Community".

pgasca replied:
Removing this comment since the standard hasn't been yet fully established.

fincd-aws committed:
Quick review of hybrid add-ons

junpengdev commented on latest/ug/nodes/hybrid-nodes-add-ons.adoc:
```diff
@@ -164,6 +178,13 @@ aws eks create-addon \
\ No newline at end of file
+[#hybrid-nodes-add-ons-metrics-server]
+=== Kubernetes Metrics Server add-on
+The control plane needs to reach Metrics Server's pod IP (or node IP if hostNetwork is enabled). Therefore, unless you run Metrics Server in hostNetwork mode, you must configure a remote pod network when creating your Amazon EKS cluster, and you must make your pod IP addresses routable.
```
Can we list some options here for making pod IP routable?

erchpm replied:
Good idea. When we call out pod routability, we should indicate that enabling BGP with your CNI is one popular way to do this.

erchpm committed:
Add routability example

Add reference to BGP for implementing pod IP routability for add-ons that require routable pod IPs.

erchpm committed:
Update hybrid-nodes-add-ons.adoc

Include CloudWatch webhook callout.

geoffcline commented:
Thanks @erchpm ! I'm merging and building now. I may make some very limited changes internally.

geoffcline commented:
Copilot Generated Summary:

This pull request includes several updates to the `latest/ug/nodes/hybrid-nodes-add-ons.adoc` file to improve clarity and provide additional information on compatible add-ons for Amazon EKS Hybrid Nodes. The most important changes include updating terminology, expanding compatibility information, and adding new sections for community add-ons.

Terminology updates:
* Updated references from "Amazon EKS add-ons from {aws}" to "{aws} add-ons" for consistency. [[1]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL14-R20) [[2]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL42-R69)

Compatibility information:
* Added a new section listing compatible community add-ons with Amazon EKS Hybrid Nodes.
* Clarified that certain add-ons have anti-affinity rules to prevent them from running on hybrid nodes.

New sections:
* Introduced a new section for the Kubernetes Metrics Server add-on, detailing the requirements for running it on hybrid nodes.

Other updates:
* Changed the headings for various add-ons to use a consistent level of subheading. [[1]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL87-R102) [[2]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL96-R120) [[3]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL123-R138) [[4]](diffhunk://#diff-a0f5cd1320f6acd01db544db29bb051658659418394b215f209a4f650f7c042eL167-R191)
